nx-dev/ui-markdoc/src/lib/nodes/fence-wrapper.component.tsx

```diff
@@ -1,9 +1,9 @@
+'use client';
 import { Fence, FenceProps } from '@nx/nx-dev/ui-fence';
-import { useRouter } from 'next/router';
+import { useRouter, usePathname } from 'next/navigation';
```

Comment:
This cannot be used in pages router. Probably need a separate wrapper or handling for this.

See: https://nx.dev/reference/project-configuration#Orchestration